c2corg_api/models/user.py

```diff
@@ -153,6 +153,53 @@ def validate_password(self, plain_password):
       unique=True)
 
 
+# Check forum_username validity with discourse
+# https://github.com/discourse/discourse/blob/master/app/models/username_validator.rb
+check_forum_username_ddl = DDL("""
+CREATE OR REPLACE FUNCTION users.check_forum_username(name TEXT)
```

Comment:
Have you tested the trigger? Or did you consider adding a unit-test for the trigger?